ParkingZoneApp/Areas/Admin/Views/ParkingSlot/Create.cshtml

```diff
@@ -0,0 +1,59 @@
+﻿@model ParkingZoneApp.ViewModels.ParkingSlotVMs.CreateVM
+@using Microsoft.AspNetCore.Identity
+@inject SignInManager<IdentityUser> SignInManager
+@{
+    ViewData["Title"] = "Create";
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+<style>
+    .checkbox {
+        transform: scale(1.5); 
+    }
+    .control-label {
+        font-size: 18px;
+    }
+</style>
+
+<h1>Create</h1>
+
+<h4>ParkingSlot</h4>
+<hr />
+<div class="row">
+    <div class="col-md-4">
+        <form asp-action="Create">
+            <input type="hidden" value="@Model.ParkingZoneId" asp-for="ParkingZoneId"/>
+            <div asp-validation-summary="ModelOnly" class="text-danger"></div>
+            <div class="form-group">
+                <label asp-for="Number" class="control-label"></label>
+                <input asp-for="Number" class="form-control" />
+                <span asp-validation-for="Number" class="text-danger"></span>
+            </div>
+            <div class="form-group">
+                <label asp-for="Category" class="control-label"></label>
+                <select asp-for="Category" class="form-control">
+                    <option value="1">Business</option>
```

Comment:
its not correct to make like that. lets suppose you added one additional category called "VIP" in this case you view will not show VIP. try to use it <select asp-for="Category" class="form-select" asp-items="Html.GetEnumSelectList<SlotCategoryEnum>()"></select>